packages/web/src/utils/third-party/cli-files/models/webpack-configs/browser.ts

```diff
@@ -5,6 +5,8 @@
  * Use of this source code is governed by an MIT-style license that can be
  * found in the LICENSE file at https://angular.io/license
  */
+// TODO: import from 'webpack' in Nx 13
+import type { Module, ModuleGraph, ChunkGraph, Chunk } from 'webpack-5';
```

Comment:
I think this would cause a problem if the user does not want webpack 5 installed. For now I think it's better to leave things untyped until Nx 13 when we remove webpack 4.

Reply:
@jaysoo it's just a type for now (`@types/webpack`), would that be a problem too? 

I have pushed 3b129c924a86753231c2223772c485aefa485213 to remove the dependency and changed to stub types. This should make it easier to swtich the types for Nx 13.